doc/src/installation.rst

```diff
@@ -69,29 +62,26 @@ All these dependencies can be installed in one command:
 
     # On apt-get based distributions
     apt-get update
-    apt-get install cmake libnetcdf-dev
+    apt-get install build-essential cmake
 
     # On yum based distribution (CentOS/RHEL)
-    yum install epel-release # The EPEL repository contains the netcdf lib
-    yum install cmake netcdf-devel
+    yum install gcc gcc-c++ make cmake
 
     # On dnf based distribution (Fedora)
-    dnf install cmake netcdf-devel
+    dnf install @development-tools cmake
 
     # On OS X with Homebrew
-    brew tap homebrew/science
-    brew install cmake netcdf
+    brew install cmake
 
-.. _NetCDF: https://www.unidata.ucar.edu/software/netcdf/
 .. _CMake: https://cmake.org/
 
 On Windows
 """"""""""
 
-You can use either MSVC 2015 compiler, or `mingw-w64`_ provided gcc. `MSYS2`_
-offer a package manager to install all the needed libraries. I recomend using it
-if you have no preference over your compiler. After the initial installation
-steps, you can run the following to install a recent C++ compiler:
+You can use either MSVC 2015 compiler, or gcc as provided by `mingw-w64`_.
+`MSYS2`_ offer a package manager to install all the needed libraries. I recomend
```

Comment:
offers

This is a matter of opinion, but given MSVC 2017 is free for community use I think it would be better to recommend it over the 3rd party `mingw-w64`.

Reply:
Yeah, this is old advice from old times =) I'll update it!

Reply:
I didn't know if it was because gcc is opensource or free!

Reply:
A bit of both, plus chemfiles was testes more with GCC than MSVC